src/components/Switch/__tests__/Switchitem.spec.js

```diff
@@ -0,0 +1,66 @@
+import '@testing-library/jest-dom';
```

Comment:
Really nitpicky: can we change the name of this file to `SwitchItem.spec.js` (instead of lowercase `i` in item)?

Reply:
Of course, thanks for catching that.